tests/polyskel_test.py

```diff
@@ -0,0 +1,237 @@
+    chk_edges = [helper_mtx2line2d(edge) for edge in chk_edges]
+
+    #TODO: temp until we updted polyskel with lb geom?
+    vertices = [(v.x,v.y) for v in polygon.vertices]
+
+    # Run function
+    if lb:
+        skel = lb_polyskel.skeletonize(vertices, [])
+    else:
+        skel = orig_polyskel.skeletonize(vertices, [])
+    tst_edges = skeleton_edges(skel)
+
+    # Tests
+    # Check types
+    is_instant = isinstance(tst_edges, list)
+    for edge in tst_edges:
+        is_instant = is_instant and isinstance(edge, LineSegment2D)
+
+    if not is_instant:
+        print('Instants not match')
+        raise Exception('Instance error.')
+
+    # Check lines
+    # Assumes order is constant
+    is_equal = True
+    for chk_edge, tst_edge in zip(chk_edges, tst_edges):
+        is_equal = is_equal and chk_edge == tst_edge
+        break
+
+    if not is_equal:
+        print('Edges not equal btwn chk_edgs:')
+        pp(chk_edges)
+        print('and actual edges:')
+        pp(tst_edges)
+        print('\nSpecifically:')
+        pp(chk_edge)
+        print('!=')
+        pp(tst_edge)
+        print('\n')
+        raise Exception('Equality error.')
+
+    return is_equal
+
+def test_polyskel_triangle():
+    """Test simplest geometry, a triangle."""
+    polygon = [
+        [0.,0.],
+        [7.,0.],
+        [4.,4.]
+    ]
+    # Make actual geom that we already sovled
+    chk_edges = [
+        [(3.8284271247461903, 1.5857864376269049), (4.0, 4.0)],
+        [(3.8284271247461903, 1.5857864376269049), (7.0, 0.0)],
+        [(3.8284271247461903, 1.5857864376269049), (0.0, 0.0)]
+    ]
+
+    assert helper_assert_polygon_equality(polygon, chk_edges)
+
+
+def test_polyskel_square():
+    """Test square."""
+    polygon = [
+        [0.,  0.],
+        [10., 0.],
+        [10.,10.],
+        [0., 10.]
+    ]
+    # Make actual geom that we already sovled
+    chk_edges = [
+        [(5.0, 5.0), (0.,   0.)],
+        [(5.0, 5.0), (0.,  10.)],
+        [(5.0, 5.0), (5.0, 5.0)],
+        [(5.0, 5.0), (10., 10.)],
+        [(5.0, 5.0), (10.,  0.)]
+    ]
+
+    assert helper_assert_polygon_equality(polygon, chk_edges)
+
+def test_polyskel_pentagon():
+    """Test concave."""
+    polygon = [
+        [0.,  0.],
+        [10., 0.],
+        [10.,10.],
+        [5., 15.],
+        [0., 10.]
+    ]
+    # Make actual geom that we already sovled
+    v1 = 7.9289321881345245
+    chk_edges = [
+        [(5.0, 7.9289321881345245), (0.,  10.)],
+        [(5.0, 7.9289321881345245), (5.0, 15.)],
+        [(5.0, 7.9289321881345245), (5., 7.9289321881345245)],
+        [(5.0, 7.9289321881345245), (10., 10.)],
+        [(5.0, 5.0),                (5., 7.9289321881345245)],
+        [(5.0, 5.0),                (10.,  0.)],
+        [(5.0, 5.0),                (0.,   0.)]
+    ]
+
+    assert helper_assert_polygon_equality(polygon, chk_edges, lb=True)
+
+# def test_polyskel_concave():
+#     """ Test rectangle with side poked in."""

```

Comment:
Why is this commented out?

Reply:
These are all failing right now, I wanted to see if the convex cases passed Travis first.

Reply:
Then let's not include them in this PR. This might be a bit too much git policing but it pays back in long term. If the code is not supposed to be part of the PR then it's better not to include it.

You can add them back with your next PR. That will also makes the relationship between the PR and tests more clear.

Reply:
Got it, I'll move them into a separate test module and avoid adding them for now.